modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -293,6 +294,12 @@ class Session implements ISession {
 
     FilePorter getFilePorter() { filePorter }
 
+    boolean publishOffload
```

Comment:
Always better declare the visibility of class attributes otherwise it's implicitly generated a getter and setter